kalite/distributed/templates/distributed/base.html

```diff
@@ -224,6 +224,9 @@
         </div> {# End wrapper #}
         {# Footer #}
         {% include "distributed/partials/_footer.html" %}
+
+        <script type="text/javascript" src="{% static 'js/tota11y.min.js' %}"></script>
```

Comment:
Ok, I get the idea... :) 
Since we do not have an `else` option here, this should be enough, right?

```
{% if settings.DEBUG %}
<script type="text/javascript" src="{% static 'js/tota11y.min.js' %}"></script>
{% endif %}
```